app/build.gradle.kts

```diff
@@ -99,6 +99,7 @@ dependencies {
     implementation(Libraries.Kotlin.stdLib)
     implementation(Libraries.appCompat)
     implementation(Libraries.ktxCore)
+    implementation(Libraries.ktxDateTime)
```

Comment:
Is ktxDataTime needed anywhere in this PR ?, if not, I think we can remove this dependency

Reply:
we're have an Instant object in MLS Protocol info that we want to show it in the group details. so yes we need it.